internal/provider/provider.go

```diff
@@ -69,9 +87,10 @@ func (p *ConductoroneProvider) Configure(ctx context.Context, req provider.Confi
 	ServerURL := data.ServerURL.ValueString()
 	ClientID := data.ClientID.ValueString()
 	ClientSecret := data.ClientSecret.ValueString()
+	TenantDomain := data.TenantDomain.ValueString()
 
 	if ServerURL == "" {
-		ServerURL = "https://{tenantDomain}.conductor.one"
+		ServerURL = fmt.Sprintf("https://%s.conductor.one", TenantDomain)
```

Comment:
please use a URL builder

Reply:
https://github.com/ConductorOne/terraform-provider-conductorone/blob/4c3c783bfa75bc61bc8a13eea3bc3803e676eab0/internal/sdk/extra_sdk_options.go#L24C14-L41

Below we call sdk.WithTenantCustom(serverURL) which called normalizeTenant(input) which uses a URL builder